config/cmake/HDF5PluginMacros.cmake

```diff
@@ -24,15 +24,9 @@ macro (EXTERNAL_PLUGIN_LIBRARY compress_type)
         URL_HASH ""
     )
   endif ()
-  FetchContent_GetProperties(PLUGIN)
-  message (VERBOSE "HDF5_INCLUDE_DIR=${HDF5_INCLUDE_DIR}")
-  if(NOT PLUGIN_POPULATED)
-    FetchContent_Populate(PLUGIN)
-    include (${HDF_RESOURCES_DIR}/HDF5PluginCache.cmake)
-    set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
-    add_subdirectory(${plugin_SOURCE_DIR} ${plugin_BINARY_DIR})
-  endif ()
-  message (VERBOSE "HDF5_INCLUDE_DIR=${HDF5_INCLUDE_DIR}")
+  include (${HDF_RESOURCES_DIR}/HDF5PluginCache.cmake)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+  FetchContent_MakeAvailable(PLUGIN)
```

Comment:
The CMake documentation says that this populates the `*_SOURCE_DIR` and `*_BINARY_DIR` variables, but does it actually add them for the build like `add_subdirectory()` does?

Reply:
yes, the make available does a add_subdirectory inside.